tests/e2e/utils.js

```diff
@@ -54,6 +57,9 @@ export async function enableExtension() {
 
   await getExtensionElement('button:enable').click();
   await expect(getExtensionElement('view:success')).toBeDisplayed();
+
+  // Give the extension some time to initialize (updating the engines in the background)
+  await browser.pause(5000);
```

Comment:
lets create a github issue to have some visual feedback once the extension is fully loaded